src/js/claims-status/routes.jsx

```diff
@@ -5,7 +5,6 @@ import YourClaimsPage from './containers/YourClaimsPage.jsx';
 import ClaimPage from './containers/ClaimPage.jsx';
 import ClaimStatusPage from './containers/ClaimStatusPage.jsx';
 import AppealStatusPage from './containers/AppealStatusPage.jsx';
-import AppealLearnMorePage from './components/AppealLearnMorePage.jsx';
```

Comment:
All of the `.jsx` suffixes on these can be removed.

Reply:
Thanks @U-DON, these have been removed.